pandas/lib.pyx

```diff
@@ -493,7 +493,12 @@ def fast_unique_multiple_list(list lists):
 
 @cython.wraparound(False)
 @cython.boundscheck(False)
-def fast_unique_multiple_list_gen(object gen):
+def fast_unique_multiple_list_gen(object gen, bint sort=True):
+    # see gh-13304
+    # this boolean was added to this function ONLY to
```

Comment:
you explanation is not required here. Just put a Parameters and explain that it would sort the keys